lib/report.c

```diff
@@ -87,6 +87,13 @@ grn_report_table(grn_ctx *ctx,
       GRN_TEXT_PUT(ctx, &description, name, name_size);
       GRN_TEXT_PUTS(ctx, &description, ">");
     }
+    if (previous_target && previous_target != table) {
+      grn_obj_unref(ctx, previous_target);
+    }
+    previous_target = target;
```

Comment:
Could you move this to `for (...)` to indicating that this is update in each loop?

Reply:
I have fixed it.